code/modules/clothing/glasses/thermal.dm

```diff
@@ -51,7 +51,6 @@
 	darkness_view = 7
 	origin_tech = list(TECH_MAGNET = 6, TECH_COVERT = 2) //add replace with Tech_Onestar when thats done made -Valo
 	spawn_tags = SPAWN_TAG_ITEM_TECH_OS
```

Comment:
```suggestion
	spawn_tags = SPAWN_TAG_TECH_OS
```

Delete the SPAWN_TAG_ITEM_TECH_OS definition from _spawner.dm since it will make the tagged items spawn in regular trash. Change instances of SPAWN_ITEM_TECH_OS to SPAWN_TECH_OS.

Reply:
OS regenerator has SPAWN_TAG_ITEM_TECH_OS, too.